inference-engine/src/plugin_api/xml_parse_utils.h

```diff
@@ -3,7 +3,8 @@
 //
 
 /**
- * @brief Basic functions to safely extract values from `pugi::xml_node` and open `pugi::xml_document`
+ * @brief Basic functions to safely extract values from `pugi::xml_node` and
```

Comment:
why this line for Plugin API is wrapped while public API is not?

Reply:
I reverted changes for public API

Reply:
but why not for all APIs?

Reply:
Fixed